isabelle-mirror/action.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+name: 'Isabelle Mirror'
+description: 'Mirrors the upstream Isabelle repository to git'
+author: Gerwin Klein <gerwin.klein@csiro.au>
```

Comment:
Do you want to update this email address?

Reply:
Thanks, I probably should, yes!